keras_cv/point_cloud/point_cloud_test.py

```diff
@@ -205,15 +203,15 @@ def test_within_box3d(self):
         assert bboxes.shape[0] == num_boxes
         assert expected_is_inside.shape[0] == num_points
         assert expected_is_inside.shape[1] == num_boxes
-        is_inside = ops.is_within_box3d(points, bboxes)
+        is_inside = point_cloud.is_within_box2d(points, bboxes)
```

Comment:
This is causing the CI failure. This should be `is_within_box3d`

Reply:
Ahhhh - thanks a lot